fulltextsearch.hh

```diff
@@ -26,7 +26,7 @@ enum {
   MaxDictionarySizeForFastSearch = 150000,
 
   // Maxumum match length for highlight search results
-  // (QWebPage::findText() crashes on too long strings)
+  // (QWebEnginePage::findText() crashes on too long strings)
   MaxMatchLengthForHighlightResults = 500
 };
```

Comment:
> this can wait for another PR

Is your "another PR" also real a fix to the issue in WebKit version? IF SO, the sooner the better! Please first do it!